index.bs

```diff
@@ -1212,6 +1189,41 @@ partial interface HTMLIFrameElement {
 
     </div>
   </section>
+  <section>
+    ## <dfn abstract-op id="check-potential-violation-in-container">Check potential violation of permissions policy in container</dfn> ## {#algo-check-potential-violation-in-container}
+
+    <div class="algorithm" data-algorithm="check-potential-violation-in-container">
+    Given a <a>navigable container</a> (|container|), this algorithm sends potential
+    violation reports.
+    1. Let |document| be |container|'s <a>node document</a>.
+    2. Let |settings| be |document|'s <a>environment settings
+       object</a>.
+    3. [=set/For each=] <a>supported feature</a> |feature|:
+        1. If the result of running <a abstract-op>Define an inherited
+           policy for feature in container at origin</a> on |feature|,
+           |container| and |container|'s <a>declared origin</a> is
+           "<code>Disabled</code>":
+          1. Let |endpoint| be the result of calling <a abstract-op>Get
+             the reporting endpoint for a feature</a> given |feature| and
+             |document|'s [=Document/permissions policy=].
+          2. Call <a abstract-op>Generate report for potential violation
+             of permissions policy on settings</a> given |feature|,
+             |settings|, "<code>Enforce</code>", and |endpoint|.
+        2. If the result of running <a abstract-op>Define an inherited
```

Comment:
I think you might want an "Else, " here  - the spec typically only sends one report per check, and the enforcement report pre-empts the report-only report.